<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationBarClass</class>
 <widget class="QWidget" name="NavigationBarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#7fa8f3;</string>
  </property>
  <widget class="QPushButton" name="preferenceSettingButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:none;color:white}
QPushButton:hover {background-color:rgb(118, 158, 226);}</string>
   </property>
   <property name="text">
    <string> 偏好设置</string>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/cross.png</normaloff>:/MainWindow/cross.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="voronoiButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:none;color:white;padding-left:14px;}
QPushButton:hover {background-color:rgb(118, 158, 226);}</string>
   </property>
   <property name="text">
    <string> Voronoi图</string>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/triangle.png</normaloff>:/MainWindow/triangle.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="resultButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:none;color:white;}
QPushButton:hover {background-color:rgb(118, 158, 226);}</string>
   </property>
   <property name="text">
    <string> 查看结果</string>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/circle.png</normaloff>:/MainWindow/circle.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>251</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>preferenceSettingButton</zorder>
  <zorder>voronoiButton</zorder>
  <zorder>resultButton</zorder>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
